includes/transformer/class-base.php

```diff
@@ -243,4 +305,69 @@ protected function get_summary_map() {
 			$this->get_locale() => $this->get_summary(),
 		);
 	}
+
+	/**
+	 * Returns the tags for the post.
+	 *
+	 * @return array The tags for the post.
+	 */
+	protected function get_tag() {
+		$tags     = array();
+		$mentions = $this->get_mentions();
+
+		if ( $mentions ) {
+			foreach ( $mentions as $mention => $url ) {
+				$tag    = array(
+					'type' => 'Mention',
+					'href' => \esc_url( $url ),
+					'name' => \esc_html( $mention ),
+				);
+				$tags[] = $tag;
+			}
+		}
```

Comment:
```suggestion
		foreach ( $mentions as $mention => $url ) {
			$tags[] = array(
				'type' => 'Mention',
				'href' => \esc_url( $url ),
				'name' => \esc_html( $mention ),
			);
		}
```
Could be simplified a little bit